.evergreen/config.yml

```diff
@@ -293,6 +293,16 @@ functions:
               fi
           fi
 
+          # Set temp credentials for AWS if python3 is available (mdb >= 3.2).
```

Comment:
Can you explain the `mdb >= 3.2` part of the comment? I don't see anything in this script invocation that's tied to server versions.

Reply:
On server versions 2.6 and 3.2 on Evergreen, the `PYTHON3_BINARY` environment variable is not set. I don't actually know why it suddenly becomes available as of `3.2`, but my guess is that Python 3 is not installed on the evg servers with these versions. `python3` is required for the `boto3` dependency. It looks like none of the CSE tests run on 2.6 or 3.2 either.

Reply:
Can you try setting `PYTHON3_BINARY` to `/usr/bin/python3` for Ubuntu 14.04? If that doesn't work, we'll have to revisit because I'd really like to keep the empty variable checks in the test files, but we'll need to do a small refactor to make that work if we can't access python3 on this OS.

Reply:
Discussed offline: setting `PYTHON3_BINARY` to `usr/bin/python3` still doesn't work. There is a python3 binary at that location for Ubuntu 14.04, but using `venv` requires the use of `apt-get` which we don't want to use. Ubuntu 14.04 is only used for 2.6 and 3.0 tasks, though, which will never need the AWS temp creds. We've decided to keep the check for `PYTHON3_BINARY` here, and I've updated the comment to convey this reasoning.